jena-tdb/src/main/java/org/apache/jena/tdb/base/block/BlockMgrCache.java

```diff
@@ -58,10 +58,7 @@ private BlockMgrCache(int readSlots, int writeSlots, final BlockMgr blockMgr)
     {
```

Comment:
Okay, I did notice that stuff (in `jena-core`) but I wanted to be sure I thoroughly understood this `jena-base` stuff before tackling it. I'm glad I did, because I clearly did _not_ understand these classes well enough to shift things onto them.